discord/channel.py

```diff
@@ -867,11 +876,26 @@ def is_nsfw(self):
             category.
         reason: Optional[:class:`str`]
             The reason for editing this channel. Shows up on the audit log.
-
+        overwrites: :class:`dict`
+            A :class:`dict` of target (either a role or a member) to
+            :class:`PermissionOverwrite` to apply to the channel.    
+            .. versionadded:: 1.3
+       
+        Example
```

Comment:
Probably should remove this example. It's too destructive to have such a simple example giving people the wrong expectation.